File: Python/algoritmo003/main.py
def executar():
    fora_de_alcance = input(
        "A lâmpada está fora de alcance? (s/n): "
    )

    if fora_de_alcance.lower() == "s":
        print("Pegar a escada.") 

    print("Pegar a lâmpada.")

    quente = input(
        "A lâmpada está quente? (s/n): "
    )

    if quente.lower() == 's':
        print("Pegar pano.")

    print("Tirar lâmpada queimada.")
    print("Colocar lâmpada boa.")

executar()

File: Java/algoritmo003/Main.java
package algoritmo003;

import java.util.Scanner;

public class Main {
    public static void executar() {

        Scanner scanner = new Scanner(System.in);

        System.out.print("A lâmpada está fora de alcance? (s/n): ");
        String foraDeAlcance = scanner.nextLine();

        if (foraDeAlcance.equalsIgnoreCase("s")) {
            System.out.println("Pegar a escada.");
        }

        System.out.println("Pegar a lâmpada.");

        System.out.print("A lâmpada está quente? (s/n): ");
        String quente = scanner.nextLine();

        if(quente.equalsIgnoreCase("s")) {
            System.out.println("Pegar pano.");
        }

        System.out.println("Tirar lâmpada queimada.");
        System.out.println("Colocar lâmpada boa.");
    }

    public static void main(String[] args) {
        executar();
    }
}